model-archiver/model_archiver/model_packaging_utils.py

```diff
@@ -217,7 +217,7 @@ def zip(export_file, model_name, model_path, files_to_exclude, manifest):
         with zipfile.ZipFile(mar_path, 'w', zipfile.ZIP_DEFLATED) as z:
             ModelExportUtils.zip_dir(model_path, z, set(files_to_exclude))
             # Write the manifest here now as a json
-            z.writestr(os.path.join(MAR_INF, MANIFEST_FILE_NAME), json.dumps(manifest, indent=4))
+            z.writestr(os.path.join(MAR_INF, MANIFEST_FILE_NAME), manifest)
```

Comment:
Looks good. Could you just add a couple of lines of how you tested this change in the description?